File: DemoMMOUnity/Assets/Game/Hoftix/Script/GenerateScripts/UIModules/LoginPanelView.cs
using ZJYFrameWork.UISerializable.UIInitView;
using UnityEngine;
using UnityEngine.UI;

namespace ZJYFrameWork.UISerializable
{
    public class LoginPanelView:UIViewInterface
    {
        public UnityEngine.UI.Image bg_Image=null;
		public UnityEngine.UI.Button Gonggao_Button=null;
		public UnityEngine.GameObject tips=null;
		public UnityEngine.UI.Text UserNameText=null;
		public UnityEngine.GameObject LoginPart=null;
		public ZJYFrameWork.UISerializable.UISerializableKeyObject LoginPart_UISerializableKeyObject=null;
		public ZJYFrameWork.UISerializable.UISerializableKeyObject RegisterPart_UISerializableKeyObject=null;
		public UnityEngine.CanvasGroup LoginStart_CanvasGroup=null;
		public ZJYFrameWork.UISerializable.UISerializableKeyObject LoginStart_UISerializableKeyObject=null;
		public ZJYFrameWork.Hotfix.UISerializable.LoginUIController LoginController=null;
		public UnityEngine.AudioSource AudioSource=null;
		


        public void Init(UIView _view)
        {
            bg_Image=_view.GetObjType<UnityEngine.UI.Image>("bg_Image");
			Gonggao_Button=_view.GetObjType<UnityEngine.UI.Button>("Gonggao_Button");
			tips=_view.GetObjType<UnityEngine.GameObject>("tips");
			UserNameText=_view.GetObjType<UnityEngine.UI.Text>("UserNameText");
			LoginPart=_view.GetObjType<UnityEngine.GameObject>("LoginPart");
			LoginPart_UISerializableKeyObject=_view.GetObjType<ZJYFrameWork.UISerializable.UISerializableKeyObject>("LoginPart_UISerializableKeyObject");
			RegisterPart_UISerializableKeyObject=_view.GetObjType<ZJYFrameWork.UISerializable.UISerializableKeyObject>("RegisterPart_UISerializableKeyObject");
			LoginStart_CanvasGroup=_view.GetObjType<UnityEngine.CanvasGroup>("LoginStart_CanvasGroup");
			LoginStart_UISerializableKeyObject=_view.GetObjType<ZJYFrameWork.UISerializable.UISerializableKeyObject>("LoginStart_UISerializableKeyObject");
			LoginController=_view.GetObjType<ZJYFrameWork.Hotfix.UISerializable.LoginUIController>("LoginController");
			AudioSource=_view.GetObjType<UnityEngine.AudioSource>("AudioSource");
			
        }
    }
}

File: DemoMMOUnity/Lua/application/app/ui/generateScripts/UIModules/LoginPanelView.lua
---@class LoginPanelView
LoginPanelView = class("LoginPanelView")
function LoginPanelView:Init(view)
	self._UIView = view:GetComponent("UIView")
	self.bg_Image = self._UIView:GetObjType("bg_Image") or CS.UnityEngine.UI.Image
	self.Gonggao_Button = self._UIView:GetObjType("Gonggao_Button") or CS.UnityEngine.UI.Button
	self.tips = self._UIView:GetObjType("tips") or CS.UnityEngine.GameObject
	self.UserNameText = self._UIView:GetObjType("UserNameText") or CS.UnityEngine.UI.Text
	self.LoginPart = self._UIView:GetObjType("LoginPart") or CS.UnityEngine.GameObject
	self.LoginPart_UISerializableKeyObject = self._UIView:GetObjType("LoginPart_UISerializableKeyObject") or CS.ZJYFrameWork.UISerializable.UISerializableKeyObject
	self.RegisterPart_UISerializableKeyObject = self._UIView:GetObjType("RegisterPart_UISerializableKeyObject") or CS.ZJYFrameWork.UISerializable.UISerializableKeyObject
	self.LoginStart_CanvasGroup = self._UIView:GetObjType("LoginStart_CanvasGroup") or CS.UnityEngine.CanvasGroup
	self.LoginStart_UISerializableKeyObject = self._UIView:GetObjType("LoginStart_UISerializableKeyObject") or CS.ZJYFrameWork.UISerializable.UISerializableKeyObject
	self.LoginController = self._UIView:GetObjType("LoginController") or CS.ZJYFrameWork.Hotfix.UISerializable.LoginUIController
	self.AudioSource = self._UIView:GetObjType("AudioSource") or CS.UnityEngine.AudioSource
end

